compiler/rustc_mir/src/const_eval/eval_queries.rs

```diff
@@ -274,6 +275,10 @@ pub fn eval_to_allocation_raw_provider<'tcx>(
                 return Err(ErrorHandled::Reported(error_reported));
             }
         }
+        let qualif = tcx.mir_const_qualif_opt_const_arg(def);
+        if qualif.error_occured {
```

Comment:
Oooh, we could make this an `Option<ErrorReported>` field instead of bool, so we can use it like the `tainted_by_errors` field and just forward the marker type